.github/workflows/verify.yaml

```diff
@@ -58,4 +62,28 @@ jobs:
       - name: "Validate Terrform format"
         working-directory: mxd
         run: |-
-          terraform fmt -check -recursive  
+          terraform fmt -check -recursive
+          
+  End-To-End-Tests:
```

Comment:
Create a separate file for backend service. `verify-backend-service.yaml`

Reply:
but the reviewer specified- "in the verify.yaml workflow"